app/config/default_config.yml

```diff
@@ -6,4 +6,7 @@ database: "local"
 userManagement: false
 sync: false
 autosave: false
+redisTimeout: 3600
```

Comment:
Let's move this to an advanced config to keep default config simple?

Reply:
Fixed